pkg/codegen/go/gen.go

```diff
@@ -507,7 +497,7 @@ func printCommentWithDeprecationMessage(w io.Writer, comment, deprecationMessage
 	}
 }
 
-func genInputInterface(w io.Writer, name string) {
+func genInputInterface(w io.Writer, name, argsName string) {
```

Comment:
Looks like the new parameter here is unused?

Reply:
Sorry it took a while to return to this. Yes - fixed now. I also fixed a niggling bug with some usage documentation generation which was causing unnecessary changes in some providers.